modules/transports/core/nhttp/src/test/java/org/apache/synapse/transport/nhttp/config/SslSenderTrustStoreHolderTest.java

```diff
@@ -0,0 +1,14 @@
+package org.apache.synapse.transport.nhttp.config;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class SslSenderTrustStoreHolderTest {
+
+    @Test
+    public void testGetInstance() {
```

Comment:
Is this test correct? If so, could you please explain what we are trying to test here?